app/konnect/gateway-manager/provision-cloud-gateway.md

```diff
@@ -0,0 +1,280 @@
+
+    1. Configure your API access as either public or private, and then click **Create Cluster**. 
+
+Your cloud gateway is now provisioned with data plane nodes. You can use it like you would any other {{site.base_gateway}} in {{site.konnect_short_name}}.
+{% endnavtab %}
+{% navtab API %}
+The {{site.konnect_short_name}} API uses [Personal Access Token (PAT)](/konnect/api/#authentication) authentication. You can obtain your PAT from the [personal access token page](https://cloud.konghq.com/global/account/tokens). The PAT must be passed in the `Authorization` header of all requests.
+
+1. Create a Dedicated Cloud Gateway control plane using the [`/control-planes` endpoint](/konnect/api/control-planes/latest/#/Control%20Planes/create-control-plane):
+    ```bash
+    curl --request POST \
+    --url https://<region>.api.konghq.com/v2/control-planes \
+    --header 'Authorization: Bearer <personal-access-token>' \
+    --header 'Content-Type: application/json' \
+    --data '{
+		"name": "cloud-gateway-control-plane",
+		"description": "A test control plane for Dedicated Cloud Gateways.",
+		"cluster_type": "CLUSTER_TYPE_CONTROL_PLANE",
+		"cloud_gateway": true,
+		"proxy_urls": [
+			{
+				"host": "example.com",
+				"port": 443,
+				"protocol": "https"
+			}
+		]
+    }'
+    ```
+	Replace the PAT as well as the following placeholders with your own values:
+	* `name`: The name that you want to display for the cloud gateway control plane.
+	* `description`: Description of the cloud gateway control plane.
+	* `host`: The URL of the host.
+
+    You should get a `201` response like the following:
+
+    ```bash
+    {
+	"id": "3e62d2e6-45e9-4046-8065-9cd28cb3487d",
+	"name": "cloud-gateway-control-plane",
+	"description": "A test control plane for Dedicated Cloud Gateways.",
+	"config": {
+		"control_plane_endpoint": "https://cf1brr4d58.us.cp0.konghq.com",
+		"telemetry_endpoint": "https://cf1brr4d58.us.tp0.konghq.com",
+		"cluster_type": "CLUSTER_TYPE_HYBRID",
+		"auth_type": "pinned_client_certs",
+		"cloud_gateway": true,
+		"proxy_urls": [
+			{
+			"host": "example.com",
+			"port": 443,
+			"protocol": "https"
+			}
+		]
+	},
+	"created_at": "2024-04-04T14:03:46.157Z",
+	"updated_at": "2024-04-04T14:03:46.157Z"
+    }
+    ```
+	Save the control plane `id` to use in the next step. 
+
+1. Create a Dedicated Cloud Gateway data plane using the [`/cloud-gateways/configurations` endpoint](link):
+
+    ```sh
+    curl --request PUT \
+	--url https://<region>.api.konghq.com/v2/cloud-gateways/configurations \
+	--header 'Authorization: Bearer <personal-access-token>' \
+	--header 'Content-Type: application/json' \
+	--data '{
+		"control_plane_id": "3e62d2c5-45e9-4046-8065-9cd28cb3487d",
+		"version": "3.6",
+		"control_plane_geo": "us",
+		"dataplane_groups": [
+			{
+				"provider": "aws",
+				"region": "ap-northeast-1",
+				"cloud_gateway_network_id": "0e465f2f-4fa3-43ed-8900-bb6210c1e554",
```

Comment:
@Guaris Was there a way that you found this in the UI? I was trying to and clicking on the network details didn't show me anything. I had to get the value using the API using the network list endpoint.